netcore/Makefile.am

```diff
@@ -57,7 +57,9 @@ link-mono:
 prepare: $(NETCORESDK_FILE) update-corefx link-mono
 
 nupkg:
-	nuget pack runtime.nuspec -properties VERSION=@VERSION@\;RID=@RID@\;PLATFORM_AOT_SUFFIX=@PLATFORM_AOT_SUFFIX@\;COREARCH=@COREARCH@
+BLAME = $(shell git blame $(top_srcdir)/configure.ac | grep AC_INIT | cut -f1 -d' ')
+VERSTUB = .$(shell git log $(BLAME)..HEAD --oneline | wc -l)
```

Comment:
nit: `git rev-list --count $(BLAME)..HEAD`

I know there are other places in the build scripts doing this, so it's probably worth a separated patch just for that.

Reply:
Interesting, I've never seen variables being defined inside of a Makefile target. Is that an automake feature?

Reply:
I do it in regular Makefiles, so I think it's quite universal. I never seem to remember the difference between `:=` and `=` though.